src/arch/arm/armv/armv7-a/cache.c

```diff
@@ -10,11 +10,6 @@
 
 #include <arch/machine/hardware.h>
 
-static inline void invalidateByWSL(word_t wsl)
```

Comment:
This is part of the cache interface and shouldn't be deleted just because it's not currently being used